### PR TITLE
Add LICENSE file to Nuget package

### LINKED ISSUE #42: Add license and 3rd party notices to nuget package
We aren't currently putting the license and 3rd party notice files in our nuget package. We should add them.

### PR DESCRIPTION
**Pull request by imback82**
Fixes #42

### CODE REVIEW

File: src/csharp/Directory.Build.props [hunk 1]
```diff
@@ -34,6 +34,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)..\..\LICENSE"
```
Comment (eerhardt) [inline src/csharp/Directory.Build.props]: I wonder if we should change the `PackageLicenseExpression` to `<PackageLicenseFile>LICENSE</PackageLicenseFile>` instead.

That way there is only 1 "truth" or "master" license - the file that is in the root of the repo. If we ever changed that file (to say Apache 2.0), the nuget package would say it is `MIT`, but this LICENSE file would say `Apache.

File: src/csharp/Directory.Build.props [hunk 2]
```diff
@@ -34,6 +34,10 @@
...
+         Pack="true"
```
Comment (eerhardt) [inline src/csharp/Directory.Build.props]: (nit) it is nice to have the attributes lined up vertically with the attribute on the first line. See the `<Content` xml below for an example.

File: src/csharp/Directory.Build.props (hunk 1 above)
Comment (imback82) [reply src/csharp/Directory.Build.props]: fixed.

File: src/csharp/Directory.Build.props (hunk 2 above)
Comment (imback82) [reply src/csharp/Directory.Build.props]: Fixed.

**Pull request merged**